appium/protocols/webdriver/can_remember_extension_presence.py

```diff
@@ -25,3 +25,8 @@ def assert_extension_exists(self: T, ext_name: str) -> T:
 
     def mark_extension_absence(self: T, ext_name: str) -> T:
         ...
+
+    def is_missing_command(self: T, error_msg: Union[str, None]) -> bool:
```

Comment:
protocols must not contain method implementations

Reply:
yea, i noticed the implementation was in webdriver.py